backend/lib/routes/terminals.js

```diff
@@ -46,7 +46,7 @@ router.route('/')
 
       const { method, params: body } = req.body
 
-      if (!_.includes(['create', 'fetch', 'list', 'config', 'remove', 'heartbeat'], method)) {
+      if (!_.includes(['create', 'fetch', 'list', 'config', 'remove', 'heartbeat', 'listProjectTerminalShortcuts'], method)) {
```

Comment:
extract constant for whiteListMethods

Reply:
this is the list of all allowed methods and the other one is a list of methods that can be called by an admin and regular users. The lists are not equal, hence it does not make sense to extract it to a constant